beta/examples/tensorflow/classification/main.py

```diff
@@ -165,14 +189,21 @@ def run(config):
 
             loss_obj = tf.keras.losses.CategoricalCrossentropy(label_smoothing=0.1)
 
+            config.nncf_config.register_extra_structs([ModelEvaluationArgs(eval_fn=model_eval_fn)])
```

Comment:
As I understand, model_eval_fn is only needed to get the FP32 baseline. For Keras API, I want to suggest considering the possibility of directly setting the FP32 baseline in the model.accuracy_aware_fit(...) and explicitly calculating it in the example.

I mean, replace the following code: 
```
#create compressed model and get the FP32 baseline metrics
def model_eval_fn(model):
        orig_model = model_fn(**model_params)
        orig_model.compile(metrics=[tf.keras.metrics.CategoricalAccuracy(name='acc@1')])
        val_x, val_y, val_sample_weight = (
            data_adapter.unpack_x_y_sample_weight(validation_dataset))
        val_logs = orig_model.evaluate(
            x=val_x,
            y=val_y,
            sample_weight=val_sample_weight,
            batch_size=None,
            steps=validation_steps,
            callbacks=None,
            return_dict=True)
        del orig_model
        gc.collect()
        return val_logs['acc@1'] * 100
		
config.nncf_config.register_extra_structs([ModelEvaluationArgs(eval_fn=model_eval_fn)])

compression_ctrl, compress_model = create_compressed_model(
	model, config.nncf_config, should_eval_original_model=is_accuracy_aware_training_mode)

# run accuracy aware fitting
train_epoch_fn, validate_fn = get_training_validation_functions(train_dataset,
	validation_dataset, callbacks, initial_epoch, validation_steps, train_steps)

# instantiate and run accuracy-aware training loop
acc_aware_training_loop = AdaptiveCompressionTrainingLoop(
    config.nncf_config, 
    compression_ctrl, 
    runner_cls=AccuracyAwareTrainingRunner
)
compress_model = acc_aware_training_loop.run(
    compress_model,
    ...
)
```
to
```
# get the FP32 baseline metrics
metrics = [tf.keras.metrics.CategoricalAccuracy(name='acc@1')]
model.compile(metrics=accuracy_aware_metrics, return_dict=True)
results = model.evaluate(validation_dataset, steps=validation_steps)

#create compressed model
compression_ctrl, compress_model = create_compressed_model(
	model, config.nncf_config, should_eval_original_model=is_accuracy_aware_training_mode)

# run accuracy aware fitting
accuracy_aware_metrics = [(tf.keras.metrics.CategoricalAccuracy(name='acc@1'), results['acc@1'])]
compress_model.accuracy_aware_fit(
    metrics=accuracy_aware_metrics,
    ...
)
```

Reply:
done